torch/testing/_internal/common_optimizers.py

```diff
@@ -401,22 +401,33 @@ def optim_inputs_func_adam(device, dtype=None):
             desc="Tensor lr with capturable and amsgrad",
         ),
     ]
-
-    total = [
-        OptimizerInput(params=None, kwargs={}, desc="default"),
-        OptimizerInput(params=None, kwargs={"lr": 0.01}, desc="non-default lr"),
-        OptimizerInput(
-            params=None, kwargs={"weight_decay": 0.1}, desc="nonzero weight_decay"
-        ),
+    mps_supported_configs = [
         OptimizerInput(
-            params=None,
-            kwargs={"weight_decay": 0.1, "maximize": True},
-            desc="maximize",
+            params=None, kwargs={"lr": torch.tensor(0.01)}, desc="Tensor lr"
         ),
-        OptimizerInput(
-            params=None, kwargs={"weight_decay": 0.1, "amsgrad": True}, desc="amsgrad"
-        ),
-    ] + (cuda_supported_configs if "cuda" in str(device) else [])
+    ]
+
+    total = (
```

Comment:
Why are the outer () included here?

Reply:
I do not know actually. That's linter's opinion :/

Reply:
ah okay, this is fine for me! changes look good from testing. i haven't thoroughly reviewed the kernel parts--deferring to @malfet and @kulinseth for those